### PR TITLE
header - adding languages support for change password alert message 

### PR DESCRIPTION
**Pull request by marwanehcine**
Adding languages support to change password alert message on header

### CODE REVIEW

File: header/src/main/resources/_header/i18n/index_es.properties [hunk 1]
```diff
@@ -11,3 +11,6 @@ admin=administración
 users=usuarios
 logout=Cerrar sesión
 login=Iniciar sesión
+remaining_days_msg_part1=solo tienes
```
Comment (fvanderbiest) [inline header/src/main/resources/_header/i18n/index_es.properties:14]: Uppercased first letter ?

**Comment by emmdurin** [comment]
Just a small change to use courtesy spanish form, if OK please squash before merge

**Comment by emmdurin** [comment]
Message rewording proposal, if OK please squash before merge

File: header/src/main/resources/_header/i18n/index_de.properties [hunk 2]
```diff
@@ -11,3 +11,6 @@ admin=Verwaltung
 users=Benutzer
 logout=Abmelden
 login=Anmelden
+remaining_days_msg_part1=Nur noch
+remaining_days_msg_part2=Tage zum
+remaining_days_msg_part3=Ändern Sie Ihr Passwort
```
Comment (tkohr) [inline header/src/main/resources/_header/i18n/index_de.properties:16]:
```suggestion
remaining_days_msg_part1=Nur noch
remaining_days_msg_part2=Tage um
remaining_days_msg_part3=Ihr Passwort zu ändern
```

**Review by emmdurin — APPROVED** [review]

File: header/src/main/resources/_header/i18n/index_de.properties (hunk 2 above)
Comment (pmauduit) [reply header/src/main/resources/_header/i18n/index_de.properties:16]: I need to refresh my german before being able to review this, but I do trust @tkohr anyway

File: header/src/main/webapp/WEB-INF/ws-servlet.xml [hunk 3]
```diff
@@ -27,8 +27,14 @@
...
-        ignore-unresolvable="true" />
+    <bean id="properties"
+          class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+        <property name="locations">
+            <list>
+                <value>file:${georchestra.datadir}/default.properties</value>
+                <value>file:${georchestra.datadir}/header/header.properties</value>
+            </list>
+        </property>
+    </bean>
```
Comment (pmauduit) [inline header/src/main/webapp/WEB-INF/ws-servlet.xml:38]: Not a blocker for me, but what the added value of defining the bean this way and not using the context: namespace ? (I'd expect the end result to be the same, but the modification surprised me.

**Review by pmauduit — APPROVED** [review]
One question remains from my side (see comments), otherwise looks good

File: header/src/main/webapp/WEB-INF/ws-servlet.xml (hunk 3 above)
Comment (marwanehcine) [reply header/src/main/webapp/WEB-INF/ws-servlet.xml:38]: In fact with context:property-placeholder, it does not work , i have tried many changes but useless.

Comment (pmauduit) [reply header/src/main/webapp/WEB-INF/ws-servlet.xml:38]: weird, it should have worked if it was defined the former way before ?! I am ok with the change anyway, as already said.

**Review by pmauduit — APPROVED** [review]

**Pull request merged**